landoapi/api/revisions.py

```diff
@@ -26,6 +27,28 @@ def get(revision_id, api_key=None):
     return _format_revision(phab, revision, include_parents=True), 200
 
 
+def land(revision_id, api_key=None):
+    """ API endpoint at /revisions/{id}/transplants to land revision. """
+    phab = PhabricatorClient(api_key)
+    revision = phab.get_revision(id=revision_id)
+    revision = _format_revision(phab, revision, include_parents=True)
+    if not revision:
+        # We could not find a matching revision.
+        return problem(
+            404,
+            'Revision not found',
+            'The requested revision does not exist',
+            type='https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404'
+        )
+
+    trans = TransplantClient()
+    id = trans.land(
+        'ldap_username@example.com', revision['repo']['url'], 'patch',
+        'destination', 'push_bookmark', 'http://pingback.url'
+    )
+    return {'request_id': id}, 202
```

Comment:
Now that I think about it, it wouldn't really hurt to return that ID, so maybe I am overthinking a small problem!